src/sentry/sentry_metrics/kafka_metrics_interface.py

```diff
@@ -0,0 +1,42 @@
+from typing import Mapping
+
+from sentry.sentry_metrics.metrics_interface import GenericMetricsBackend
+from sentry.sentry_metrics.use_case_id_registry import UseCaseID
+
+
+class KafkaGenericMetricsBackend(GenericMetricsBackend):
```

Comment:
IMO we probably shouldn't create this class (the Kafka-specific one) until we've written the logic for actually populating the topic with data

Reply:
Ah, that's true. I can remove it